### DISCUSSION TITLE
Video on beatmaps being filled to the screen instead of fitting the screen

### DISCUSSION THREAD

**Discussion by 1ces0n1c** [ppy/osu#17834] [Q&A]
On maps that have video with small resolution and a different aspect ratio the video fills itself instead of fitting the screen so some of the top and bottom get cut off

**Comment by bdach** [comment]
This is intentional behaviour. See https://github.com/ppy/osu/issues/12291

**Reply by 1ces0n1c** [reply]
If for some fitting just the width is a problem, but for others fitting both is a problem, then why couldn't there be an option to have it fit the width and height?
It was like that on stable with the beatmaps i have this issue.
And there is a map that has the same video resolution and ratio as the ones i have problem with but on stable it fits just the width but on lazer it fits both.
